src/BulletGauge/BulletGauge.tsx

```diff
@@ -0,0 +1,81 @@
+import { BulletGaugeProps } from "./BulletGauge.types";
+import Plotly from "react-plotly.js";
+import React from "react";
+import { useTheme } from "@mui/material";
+
+const BulletGauge = ({ title, value, suffix }: BulletGaugeProps) => {
```

Comment:
Missing JSDoc comments for the component. https://gist.github.com/mattcorner/cf4bfb4cd967bb6fe13d2433010d516a#file-example-tsx-L37-L47